app/models/concerns/push_updates_for_subscription.rb

```diff
@@ -21,8 +21,8 @@ def send_new_message_notification
   def app_name
     related_object = message.related_object
     klass_name = related_object.class.name
-    created_by_id = related_object.created_by_id
-    if klass_name == 'Order'
+    created_by_id = related_object.try(:created_by_id)
+    if ['Order', 'Package'].include?(klass_name)
```

Comment:
same as per comment for `push_updates_for_message`. we just want to send package updates to the Stock app